### PR TITLE
Remove ONTIMEINTERVAL subscriptions

### PR DESCRIPTION
**Pull request by fgalan**
@kzangeli 
@crbrox 

After 0.28.0 gets released a rebase/merge from develop is needed. After that, the following items are needed to complete this PR:

* Add CNR entry
* Modify deprecated.md file to add the final release date for 0.28.0 in the table.

### CODE REVIEW

File: doc/manuals/deprecated.md [hunk 1]
```diff
@@ -17,12 +17,12 @@ A list of deprecated features and the version in which they were deprecated foll
...
-* ONTIMEINTERVAL subscriptions are deprecated since Orion 0.26.0. ONTIMEINTERVAL subscriptions have
-  several problems (introduce state in CB, thus making horizontal scaling configuration much harder,
-  and makes it difficult to introduce pagination/filtering). Actually, they aren't actually needed,
-  as any use case based on ONTIMEINTERVAL notification can be converted to an equivalent use case
-  in which the receptor runs queryContext at the same frequency (and taking advantage of the
-  features of queryContext, such as pagination or filtering).
+* ONTIMEINTERVAL subscriptions are deprecated since Orion 0.26.0 (removed in Orion 1.0.0).
+  ONTIMEINTERVAL subscriptions have several problems (introduce state in CB, thus making horizontal
+  scaling configuration much harder, and makes it difficult to introduce pagination/filtering).
+  Actually, they aren't actually needed, as any use case based on ONTIMEINTERVAL notification can
```
Comment (kzangeli) [inline doc/manuals/deprecated.md]: Actually, they aren't actually ...
Pick one :-)

File: src/lib/mongoBackend/MongoGlobal.cpp [hunk 2]
```diff
@@ -3004,9 +2863,9 @@ BSONArray processConditionVector
         *notificationDone = true;
       }
     }
-    else  // ON_VALUE_CONDITION
+    else
     {
-      // FIXME: not implemented
+      LM_E(("Runtime Error (unknown condition type: %s)", nc->type.c_str()));
```
Comment (kzangeli) [inline src/lib/mongoBackend/MongoGlobal.cpp]: '%s' - always nice to have the quotes ..

Comment (fgalan) [inline src/lib/mongoBackend/MongoGlobal.cpp]: Fixed in 90a0b9d

File: doc/manuals/deprecated.md (hunk 1 above)
Comment (fgalan) [inline doc/manuals/deprecated.md]: Fixed in 90a0b9d

**Comment by kzangeli** [comment]
LGTM

**Comment by crbrox** [comment]
LGTM

**Pull request merged**